src/extensions/locales-defaults/locales/en.json

```
@@ -24,14 +26,18 @@
         "menu.showSideBar.label": "Toggle Side Bar Visibility",
         "menu.showStatusBar": "Show Status Bar",
         "menu.showActivityBar": "Show Activity Bar",
+        "menu.hideActivityBar": "Hide Activity Bar",
         "menu.showPanel": "Show Panel",
         "menu.showPanel.title": "Toggle Panel",
+        "menu.defaultProjectName": "Default Project Name",
         "menu.run": "Run",
+        "menu.runTask": "Run Task",
         "menu.help": "Help",
+        "menu.about": "About",
         "menu.menuBarHorizontal": "Menu Bar Horizontal Mode",
         "menu.menuBarVertical": "Menu Bar Vertical Mode",
-        "sidebar.explore.title": "Explorer",
         "sidebar.explore.folders": "Folders",
+        "sidebar.explore.title": "Explore",
```

```suggestion
        "sidebar.explore.title": "Explorer",
```
Keep the same abstraction with VS Code.

---

Good!. My mistake.